src/test/java/org/primeframework/mvc/PrimeBaseTest.java

```diff
@@ -123,6 +124,8 @@ public abstract class PrimeBaseTest {
 
   @Inject public ObjectMapper objectMapper;
 
+  @Inject public Encryptor encryptor;
```

Comment:
I'm open to ideas here, but on CleanSpeak, we end up with these massive base test classes with lots of injected fields. I've tried to move the field closer to the test class using it so that doesn't happen. It also makes it easier to see which tests are using what to me.

But I suppose we are also using this in multiple inherited test classes as well.

Your call. Food for thought.

Reply:
Pros and cons as you said. When I got to four classes I was injecting the object, I moved it to the base class, but it does overexpose `Encryptor` compared to where it's needed. It's currently only used in four classes out of 59 that extend from `BasePrimeTest`. We can hold off on polluting the base class for now 😉